core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedDocServiceTest.java

```diff
@@ -307,6 +309,19 @@ private static void addJsonMethodInfo(Map<Class<?>, Set<MethodInfo>> methodInfos
         methods.add(methodInfo2);
     }
 
+    private static void addPeriodMethodInfo(Map<Class<?>, Set<MethodInfo>> methodInfos) {
```

Comment:
Do we need to add DocService test?
I think we can test the `Period` converter by registering an annotated service that takes a Period as a `@Param` and calling it by a `WebClient`.

For example:
https://github.com/line/armeria/blob/52fd3612fd38a7f5c8c2da235b8cc65e15bfd702/core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceRequestConverterTest.java#L917-L923

Reply:
I added DocService test because @minwoox's [suggestion](https://github.com/line/armeria/issues/2760#issuecomment-637889305). (Did I understand wrong?)

And I add test case for converting. Thank you for kind instructions.

Reply:
I see. I missed the comment.

Reply:
What I meant was that we need to check whether the type is rendered well or not on the DocService page.
I have checked it and it seems OK. Thanks!

Reply:
I see. Thanks!